Lock bruno/cli verison

contrun opened the pull request:
The hack in https://github.com/nervosnetwork/cfn-node/blob/f740c4b54570162dd509f98ef38c53b895725732/tests/bruno/e2e/open-use-close-a-channel/06-generate-a-few-blocks.bru no longer works. We will have a

> Too many jumps, possible infinite loop

error in the newest version (1.20.1) of bruno. For now, we just lock bruno to an older version. 1.20.0 is the newest working version.

contrun committed:
Lock bruno/cli verison

The hack in
`./tests/bruno/e2e/open-use-close-a-channel/06-generate-a-few-blocks.bru`
no longer works in newest bruno version.
We will have a

> Too many jumps, possible infinite loop

error in the newest version (1.20.1) of bruno.
For now, we just lock bruno to an older version.
1.20.0 is the newest working version.

quake commented:
we may use [generate_epochs](https://github.com/nervosnetwork/ckb/tree/develop/rpc#integration_test-generate_epochs)

contrun committed:
Use generate epochs instead of generate blocks

contrun committed:
Use bruno 1.20.0 as setVar broken

Even if we set `CHANNEL_ID` we are stilling sending
messages like
{"id":"42","jsonrpc":"2.0","method":"add_tlc","params":[{"channel_id":"{{CHANNEL_ID}}","amount":"0x5f5e100","payment_hash":"0x266cec97cbede2cfbce73666f08deed9560bdf7841a7a5a51b3a3f09da249e21","expiry":40}]}
where `{{CHANNEL_ID}}` is not replaced.
